source/features/mark-merge-commits-in-list.tsx

```diff
@@ -38,6 +38,10 @@ export function getCommitHash(commit: HTMLElement): string {
 	return $('a.markdown-title', commit)!.pathname.split('/').pop()!;
 }
 
+export function getFeatureCommitHash(commit: HTMLElement): string {
+	return $('.markdown-title a', commit)!.pathname.split('/').pop()!;
```

Comment:
You can move this function to `source/features/tags-on-commits-list.tsx` because it's only used there

Reply:
I think #7003 will use it, so I will send a new pr for #7003 after you merge this pr.

Reply:
You can open both PRs at the same time. If you keep the function in the same file, git automatically resolves the conflict.